### DISCUSSION TITLE
Cmake not found

### DISCUSSION THREAD

**Discussion by cramile** [xmake-io/xmake#3800] [Q&A]
抱歉，内网电脑没有连外网，只能手机拍屏了
Lua代码如下：
![IMG_20230601_112950](https://github.com/xmake-io/xmake/assets/13780433/18c58494-b4a8-4004-b045-4a06fb1c42c9)

报错内容如下：
![16855903550596241506127858434500](https://github.com/xmake-io/xmake/assets/13780433/c5398916-c92d-4e7c-ab86-ef84f75c7e1b)

我在_load_package(packagename, requireinfo, opt)函数中写了部分print，输出参数信息如下
![16855903941404184570973031248887](https://github.com/xmake-io/xmake/assets/13780433/7b769651-43c2-420b-b50b-6aa5473cc979)

Cmake和xmake的版本如下：
![16855904987584948558824384138650](https://github.com/xmake-io/xmake/assets/13780433/7e4cdb81-5a8e-44b4-ba04-432d54006109)

**Comment by waruqi** [comment]
先执行 `xmake l find_package cmake` 给我看下输出。。

然后删了你自己加的这些 logs 
执行

```
xmake f -cvD
```

给我看下完整 logs

**Reply by cramile** [reply]
![1685591702837613746391768591599](https://github.com/xmake-io/xmake/assets/13780433/6c41b6f1-8118-43ed-92d9-de93fdf5a32b)

**Reply by waruqi** [reply]
因为你没连外网，没有拉取到 xmake-repo 仓库，所以找不到 cmake 包。。

至少需要先拉取到 xmake-repo 仓库才行。。或者 `add_deps("cmake", {system = true})` 也改成系统的 试试
